src/main/java/org/sonarsource/sonarlint/ls/connected/SecurityHotspotsHandlerServer.java

```diff
@@ -216,7 +234,7 @@ public void handle(ClassicHttpRequest request, ClassicHttpResponse response, Htt
         // Ignored
       }
       if (!params.containsKey("server") || !params.containsKey("project") || !params.containsKey("hotspot")) {
-        response.setCode(HttpURLConnection.HTTP_BAD_REQUEST);
+        response.setCode(HttpStatus.SC_BAD_REQUEST);
```

Comment:
Why change from java.net to apache?

Reply:
Since we are using the Apache HttpClient, I feel it was more consistent.